cmd/up/forwards.go

```diff
@@ -105,5 +118,89 @@ func (up *upContext) sshForwards(ctx context.Context) error {
 		return fmt.Errorf("failed to add entry to your SSH config file")
 	}
 
-	return up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
+	err := up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
+	if err != nil {
+		return err
+	}
+
+	if isNeededGlobalForwarder(up.Manifest.GlobalForward) {
+		up.GlobalForwarderStatus = make(chan error, 1)
+		go up.setGlobalForwardsIfRequiredLoop(ctx)
+	}
+
+	return nil
+}
+
+func (up *upContext) setGlobalForwardsIfRequiredLoop(ctx context.Context) {
+	ticker := time.NewTicker(time.Second)
+
+	for {
+		if !isNeededGlobalForwarder(up.Manifest.GlobalForward) {
+			return
+		}
+
+		select {
+		case <-ticker.C:
+			err := addGlobalForwards(up)
+			if err != nil {
+				up.GlobalForwarderStatus <- err
+				return
+			}
+
+			err = up.Forwarder.StartGlobalForwarding()
+			if err != nil {
+				up.GlobalForwarderStatus <- err
+				return
+			}
+		case <-ctx.Done():
```

Comment:
When is this context signaled to be done? We need to double check that we are gracefully closing this loop.

Reply:
graceful shutdown is handled by _activate_ method https://github.com/okteto/okteto/blob/6c7d7d1d9b067e3b68c78f443e13387c4806cfa7/cmd/up/activate.go#L49 

this context is signaled to be done when starts _shutdown_ sequence.
https://github.com/okteto/okteto/blob/6c7d7d1d9b067e3b68c78f443e13387c4806cfa7/cmd/up/up.go#L743-L746